### ISSUE TITLE
furtive.co domain has expired

### ISSUE THREAD

**Issue by timbotron** [johno/furtive#69]
Hello, It looks like the furtive.co domain has expired, just FYI!

<img width="1028" alt="screen shot 2019-02-18 at 1 59 50 pm" src="https://user-images.githubusercontent.com/283604/52978289-7b12b000-3385-11e9-829c-6905d017197a.png">

I am really loving furtive!

**Comment by johno** [comment]
Whoops! I just renewed the domain, my bad. It should be live in the next 15 mins or so.

**Comment by Rudloff** [comment]
It is still expired :slightly_frowning_face:

**Comment by timbotron** [comment]
oh noes.

<img width="914" alt="Screen Shot 2019-05-06 at 3 41 09 PM" src="https://user-images.githubusercontent.com/283604/57259715-66a19380-7015-11e9-96fb-91e5946cd488.png">

**Comment by adamsanderson** [comment]
Heads up: It's now a site pedaling malware.
So goes the internet.

**Comment by Rudloff** [comment]
I think you should at least removing it from the README and the Github description :slightly_frowning_face:

**Comment by inspire22** [comment]
Yeah; hopefully we can set up a mirror somewhere, since the docs were there it's kind hard to use this now, and it was my favorite framework :(

**Comment by johno** [comment]
Yeah... Unfortunately I can't get the domain back 😞, but I'll get a new docs deployment together this week that I can link to.

**Comment by warwickgrigg** [comment]
This is a useful workaround in the meantime: https://rawcdn.githack.com/johno/furtive/36235167f90b6614dc3e6c9ba806ded3e23ea567/index.html

**Comment by johno** [comment]
Looks like whoever registered the domain is now pointing it to the gh pages deployment 🤷
